content/en/getting_started/tagging/unified_service_tagging.md

```diff
@@ -294,116 +294,121 @@ To form a single point of configuration for all telemetry emitted directly from
+   **Note**: The Datadog DogStatsD clients for .NET and PHP do not yet support this functionality.
 
-##### Single service per host
+[1]: /metrics/
+   {{% /tab %}}
 
-Set the following configuration in the Agent's [main configuration file][1]:
+   {{% tab "System Metrics" %}}
 
-```yaml
-env: <ENV>
-tags:
-    - service:<SERVICE>
-```
+   You can add `env` and `service` tags to your infrastructure metrics.
 
-This setup guarantees consistent tagging of `env` and `service` for all data emitted by the Agent.
+   The tagging configuration for service metrics lives closer to the Agent in non-containerized contexts.
+   Given that this configuration does not change for each invocation of a service's process, adding `version`
+   to the configuration is not recommended.
 
-##### Multiple services per host
+   #### Single service per host
 
-Set the following configuration in the Agent's [main configuration file][1]:
+   Set the following configuration in the Agent's [main configuration file][1]:
 
-```yaml
-env: <ENV>
-```
+   ```yaml
+   env: <ENV>
+   tags:
+       - service:<SERVICE>
+   ```
 
-To get unique `service` tags on CPU, memory, and disk I/O metrics at the process level, configure a [process check][2] in the Agent's configuration folder (for example, in the `conf.d` folder under `process.d/conf.yaml`):
+   This setup guarantees consistent tagging of `env` and `service` for all data emitted by the Agent.
 
-```yaml
-init_config:
-instances:
-    - name: web-app
-      search_string: ["/bin/web-app"]
-      exact_match: false
-      service: web-app
-    - name: nginx
-      search_string: ["nginx"]
-      exact_match: false
-      service: nginx-web-app
-```
+   #### Multiple services per host
+
+   Set the following configuration in the Agent's [main configuration file][1]:
+
+   ```yaml
+   env: <ENV>
+   ```
+
+   To get unique `service` tags on CPU, memory, and disk I/O metrics at the process level, configure a [process check][2] in the Agent's configuration folder (for example, in the `conf.d` folder under `process.d/conf.yaml`):
+
+   ```yaml
+   init_config:
+   instances:
+       - name: web-app
+         search_string: ["/bin/web-app"]
+         exact_match: false
+         service: web-app
+       - name: nginx
+         search_string: ["nginx"]
+         exact_match: false
+         service: nginx-web-app
+   ```
 
-**Note**: If you already have a `service` tag set globally in your Agent's main configuration file, the process metrics are tagged with two services. Since this can cause confusion with interpreting the metrics, it is recommended to configure the `service` tag only in the configuration of the process check.
+   **Note**: If you already have a `service` tag set globally in your Agent's main configuration file, the process metrics are tagged with two services. Since this can cause confusion with interpreting the metrics, it is recommended to configure the `service` tag only in the configuration of the process check.
```

Comment:
```suggestion
   **Note**: If you already have a `service` tag set globally in your Agent's main configuration file, the process metrics are tagged with two services. Since this can cause confusion when you are interpreting the metrics, it is recommended that you configure the `service` tag only in the configuration of the process check.
```